spec/requests/dashboard_spec.rb

```diff
@@ -39,6 +39,19 @@
         expect(response.body).to include "Delete"
       end
 
+      it "renders subscriptions for articles with subscriptions" do
+        user.add_role(:admin) # TODO: (Alex Smith) - update roles before release
```

Comment:
One suggestion, we have had some issues with flakiness around roles in specs, one way to avoid this would be to stub the role so `user.admin?` returns true rather than adding the role itself. This would also help speed up the specs. Just a thought.

Reply:
The roles are going to change in the next PR, so I'll address it there 👍 .